src/Milky/Models/OutputSettings.cs

```diff
@@ -37,5 +38,15 @@ public class OutputSettings
         /// Console output color for <see cref="ComboResult.Invalid"/>
         /// </summary>
         public ConsoleColor InvalidColor { get; set; } = ConsoleColor.Red;
+
+        /// <summary>
+        /// Merge outputs to global results file
+        /// </summary>
+        public bool GlobalOutput { get; set; } = false;
+
+        /// <summary>
+        /// Custom ConsoleColor output if capture conditional check is true
+        /// </summary>
+        public Dictionary<ConsoleColor, KeyValuePair<string, Func<object, bool>>> SpecialColors { get; set; } = new Dictionary<ConsoleColor, KeyValuePair<string, Func<object, bool>>>();
```

Comment:
Maybe we can use a `Predicate<object>` here instead of a `Func<object, bool>`, I prefer it